spec/lib/index/index_spec.rb

```diff
@@ -71,8 +71,8 @@
 
   it "correctly sets hosts" do
     expect(index.client.send(:hosts)).to eql([
-      {"host" => "localhost", "port" => 9200, "protocol" => "http"},
-      {"host" => "127.0.0.1", "port" => 9200, "protocol" => "http"}
+      {host: "localhost", port: 9200, protocol: "http"},
+      {host: "127.0.0.1", port: 9200, protocol: "http"}
```

Comment:
I don't really know why we're testing host setting here, but I didn't want to conflate removing potentially unnecessary specs with first getting out CI to be green.